libs/langgraph/src/tests/pregel.test.ts

```diff
@@ -3541,6 +3541,7 @@ export function runPregelTests(
           hello: "there",
           bye: "world",
           messages: ["hello"],
+          // @ts-expect-error This should emit a TS error
```

Comment:
We can now actually catch excessive keys!

Reply:
ha nice!